apps/metadata-editor/src/app/edit/edit-page.component.html

```diff
@@ -1,11 +1,38 @@
-<div class="flex flex-col h-full">
-  <div class="w-full h-auto shrink-0">
-    <md-editor-top-toolbar></md-editor-top-toolbar>
-  </div>
-  <div class="grow overflow-auto relative">
-    <div class="absolute top-0 left-0 w-2/3 z-10 pointer-events-none">
-      <gn-ui-notifications-container></gn-ui-notifications-container>
+<ng-container *ngIf="fields$ | async as fields">
+  <div class="flex flex-col h-full">
+    <div class="w-full h-auto shrink-0">
+      <md-editor-top-toolbar></md-editor-top-toolbar>
+      <md-editor-breadcrumbs
+        [pages]="fields.pages"
+        [selectedPage]="selectedPage"
+        (selectedPageChange)="selectedPageChange($event)"
+      ></md-editor-breadcrumbs>
+    </div>
+    <div class="grow overflow-auto relative">
+      <div class="absolute top-0 left-0 w-2/3 z-10 pointer-events-none">
+        <gn-ui-notifications-container></gn-ui-notifications-container>
+      </div>
+      <gn-ui-record-form
+        [page]="fields.pages | find: selectedPage"
```

Comment:
I'm not sure that this custom pipe really brings value; having a `selectedPage` getter on the component should be less complex overall (less code)